astropy/utils/data.py

```diff
@@ -903,18 +903,7 @@ def get_free_space_in_dir(path):
         is on.
     """
 
-    if sys.platform.startswith('win'):
-        import ctypes
-        free_bytes = ctypes.c_ulonglong(0)
-        retval = ctypes.windll.kernel32.GetDiskFreeSpaceExW(
-                ctypes.c_wchar_p(path), None, None, ctypes.pointer(free_bytes))
-        if retval == 0:
-            raise OSError('Checking free space on {!r} failed '
-                          'unexpectedly.'.format(path))
-        return free_bytes.value
-    else:
-        stat = os.statvfs(path)
-        return stat.f_bavail * stat.f_frsize
+    return shutil.disk_usage(path).free
```

Comment:
Unrelated.

Reply:
I don't know how that got in there - it's from PR #9182 . I'll clear it out.